src/app/mainPage/rankingTable/RankingTable.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { translations } from 'i18n/messages';
+import { render, screen } from 'tests';
+import { RANKIG_MOCK } from 'tests/mock/rankingMock';
+import { RankigTable } from './RankingTable';
+
+describe('RankingTable component', () => {
+  it('renders ranking table correctly', () => {
```

Comment:
A propos tego testu, właściwie to częścią logiki jest to, że czasem dodawane jest ` km`. Może to też sprawdzić?